src/services/crons/aws.query.cache.warmer.service.ts

```diff
@@ -104,6 +104,31 @@ export class AWSQueryCacheWarmerService {
             ]);
             await this.deleteCacheKeys(cachedKeys);
         }
+
+        const allTokensVolumeUSDCompleteValuesSum = await this.analyticsQuery.getSumCompleteValues({
+            series: '%-%',
```

Comment:
we should use pairs volumes for total volumes `erd1%`

Reply:
updated